src/OpenTelemetry.Instrumentation.StackExchangeRedis/TracerProviderBuilderExtensions.cs

```diff
@@ -78,7 +78,7 @@ public static class TracerProviderBuilderExtensions
             TracerProviderBuilder builder,
             IConnectionMultiplexer connection,
             StackExchangeRedisCallsInstrumentationOptions options,
-            Action<StackExchangeRedisCallsInstrumentationOptions> configure = null)
+            Action<StackExchangeRedisCallsInstrumentationOptions> configure)
```

Comment:
This is a breaking change. The default `null` value is there as often the user is OK with the default `StackExchangeRedisCallsInstrumentationOptions` value (pardon my English).